package.json

```
@@ -15,15 +15,16 @@
   "license": "MIT",
   "devDependencies": {
     "babel": "^6.23.0",
-    "electron": "^18.2.3",
-    "webpack": "^5.6.0"
+    "electron": "^18.3.15",
+    "webpack": "^5.74.0"
```

```suggestion
    "webpack": "^5.74.0"
```

Don't think webpack is actually used in this